### ISSUE TITLE
Splatmaps

### ISSUE THREAD

**Issue by Gusher** [Zylann/godot_heightmap_plugin#253]
When exporting does it also save the splatmap / blendmap, RGBA?

**Comment by Zylann** [comment]
Yes, all the maps are there. The plugin doesn't remove things on export.

**Comment by Gusher** [comment]
Do I need to compile the plugin?

**Comment by Zylann** [comment]
No, it's using GDScript so there is no compilation needed. There is a GDNative component that is already precompiled for Linux and Windows, and on other platforms it fallbacks on GDScript.

**Comment by Gusher** [comment]
Awesome.. thanks
